### ISSUE TITLE
[BUG] Could not check for peer dependencies of x package

### ISSUE THREAD

**Issue by hoang-innomizetech** [flowaccount/nx-plugins#36]
I am getting below warning when deploying our service

```
getting external modules
create a package.json with first level dependencies
Serverless: WARNING: Could not check for peer dependencies of serverless-mysql
generate lock entry for yarn for dependency graph to work.
get the packagelist with dependency graph and depth=2 level
re-writing package.json with dependency-graphs
Serverless: WARNING: Could not determine version of module tslib
Serverless: WARNING: Could not check for peer dependencies of serverless-mysql
run packager to  install node_modules
```

Here is the sample code of my Lambda function

```
// Require and initialize outside of your main handler
import * as serverlessMysql from 'serverless-mysql';
const mysql = serverlessMysql({
  config: {
    host     : process.env.ENDPOINT,
    database : process.env.DATABASE,
    user     : process.env.USERNAME,
    password : process.env.PASSWORD
  }
})

// Main handler function
export const handler = async (event, context) => {
  // Run your query
  let results = await mysql.query('SELECT * FROM table')

  // Run clean up function
  await mysql.end()

  // Return the results
  return results
}
```

I have debugged and figured out that the modulePackagePath is invalid (the root dir is duplicated in the path), see below example

modulePackagePath /Users/hoangle/Documents/Working/Source/my-sls-api/Users/hoangle/Documents/Working/Source/my-sls-api/node_modules/serverless-mysql/package.json

**Comment by wickstargazer** [comment]
Can i see your serverless.yml

Also are you trying to deploy by function or the whole thing?

**Comment by hoang-innomizetech** [comment]
The serverless.yml is straightforward

```
service: myapi
frameworkVersion: ">=1.1.0 <2.0.0"
plugins:
  - serverless-offline
package:
  individually: true
  excludeDevDependencies: true
  custom:
    enable_optimize:
      local: false
provider:
  name: aws
  runtime: nodejs12.x
  stage: dev
  region: us-east-1
  profile: recom_dev

functions:
  hello-world:
    handler: src/hello-world/hello-world.handler
    events:
      - http:
          path: hello-world
          method: get


```

The workspace.json also default

**Comment by hoang-innomizetech** [comment]
@wickstargazer today I also found a mistake of the normalize util at https://github.com/flowaccount/nx-plugins/blob/56f6d93115171396fcffd9350e59bc4079a6ee2e/libs/nx-serverless/src/utils/normalize.ts#L270

I can submit a PR that fixes both issues. Can I have write access to this repo?

**Comment by wickstargazer** [comment]
Hi! Would you mind forking it and submitting a PR?

**Comment by hoang-innomizetech** [comment]
I have submitted a PR to fix two issues, let me know if I need to provide more details

**Comment by hoang-innomizetech** [comment]
There are two issues on the normalize util
1. The packagePath is a full path, so we can use dirname to get workspace root path
2. Missing the packageJson when calling the getProdModules function

**Comment by wickstargazer** [comment]
Hi!, I have published `0.5.1` please see if it works :)
